tumblr_client.gemspec

```diff
@@ -4,14 +4,14 @@ require File.join(File.dirname(__FILE__), 'lib/tumblr/version')
 Gem::Specification.new do |gem|
   gem.add_dependency 'faraday', '~> 0.9.0'
   gem.add_dependency 'faraday_middleware', '~> 0.9'
-  gem.add_dependency 'json'
-  gem.add_dependency 'simple_oauth'
-  gem.add_dependency 'oauth'
-  gem.add_dependency 'mime-types'
-  gem.add_development_dependency 'rake'
-  gem.add_development_dependency 'rspec'
-  gem.add_development_dependency 'webmock'
-  gem.add_development_dependency 'simplecov'
+  gem.add_dependency 'json', '~> 1.8'
+  gem.add_dependency 'simple_oauth', '~> 0.3'
+  gem.add_dependency 'oauth', '~> 0.5'
+  gem.add_dependency 'mime-types', '~> 2.99'
```

Comment:
The usage of Mime::Types in this library is so simple it is perfectly compatible with both 2.x and 3.x, so I believe this is unnecessary.  Since the majority seems to have migrated to version 3 already and support for mime-types 2.x is now dated (EOL scheduled on 2017-11-17) I'd appreciate if this constraint could be relaxed (like `['>= 2.99', '< 4']`) or just removed.